src/materialized/src/lib.rs

```diff
@@ -174,6 +177,13 @@ pub enum TlsMode {
     },
 }
 
+#[derive(Debug, Clone)]
+pub struct MzCloudParams {
+    pub admin_api_token_url: String,
+    pub jwk: Vec<u8>,
+    pub tenant_id: String,
+}
```

Comment:
Hmm, why this separate type? Can't we just embed the `MzCloudAuthentication` in the config directly?

Reply:
Done.